Gemfile

```diff
@@ -23,6 +22,9 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'bootstrap-sass', '~> 3.2.0'
```

Comment:
I'm not a great front-end person :(
Can you explain the difference between Sass and CSS? Is Sass easy to learn for people who only know CSS?